components/plan-build/bin/composite_build_functions.bash

```diff
@@ -114,8 +115,8 @@ _resolve_all_exports() {
 
   for rs in "${!resolved_services[@]}"; do
     resolved=${resolved_services[$rs]}
-    exports=("$(_exports_for_pkg ${resolved})")
-    pkg_export_map[$resolved]="${exports[@]}"
+    exports=("$(_exports_for_pkg "${resolved}")")
+    pkg_export_map[$resolved]="${exports[*]}"
```

Comment:
This is definitely incorrect with the `@` (see https://github.com/koalaman/shellcheck/wiki/SC2124), but I'd like some confirmation about the desired behavior to know whether this change is the proper fix.

Reply:
I think this is the right thing... If I remember correctly, it needs to be a space-separated string, like `"foo bar baz"`.